src/components/title/_fields/edition/edition-field.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Field } from 'redux-form';
+
+import { TextField } from '@folio/stripes-components';
+import styles from './edition-field.css';
+
+export default function EditionField() {
+  return (
+    <div
+      data-test-eholdings-edition-field
+      className={styles['edition-field']}
+    >
+      <Field
+        name="edition"
+        component={TextField}
+        label="Edition"
+      />
+    </div>
+  );
+}
+
+export function validate(values) {
+  const errors = {};
+
+  if (values.edition && values.edition.length > 250) {
```

Comment:
Do we want to also validate that they didn't pass an empty string?

`"    "` would pass this validation for example